.gitignore

```diff
@@ -1,10 +1,16 @@
-node_modules
```

Comment:
shouldn't we keep `node_modules` here?

Reply:
There are no modules. https://github.com/Kwenta/margin-manager/blob/v2.0.0/package.json